plutus-script-utils/src/Plutus/Script/Utils/V1/Typed/Scripts/Validators.hs

```diff
@@ -0,0 +1,53 @@
+{-# LANGUAGE AllowAmbiguousTypes #-}
+{-# LANGUAGE DataKinds           #-}
+{-# LANGUAGE DerivingStrategies  #-}
+{-# LANGUAGE TypeFamilies        #-}
+
+{-# OPTIONS_GHC -fno-specialise #-}
+{-# OPTIONS_GHC -Wno-simplifiable-class-constraints #-}
+{-# OPTIONS_GHC -fno-omit-interface-pragmas #-}
+
+module Plutus.Script.Utils.V1.Typed.Scripts.Validators
+    ( UntypedValidator
+    , mkUntypedValidator
```

Comment:
Isn't it strange that we provide `Untyped` things from the `Typed` submodules? Shouldn't they belong to `Untyped`?

Reply:
Mmm... good point. I guess we could put it in `Plutus.Script.Utils.V1.Scripts`.

Reply:
Can you confirm that the new modules make sense?

Reply:
Yeah, looks good 👍